src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -1025,11 +1025,14 @@ def update_resource_group(instance, tags=None):
 
 
 def export_group_as_template(
-        cmd, resource_group_name, include_comments=False, include_parameter_default_value=False):
+        cmd, resource_group_name, resource_ids=None, include_comments=False, include_parameter_default_value=False, skip_resource_name_parameterization=False, skip_all_parameterization=False):
```

Comment:
`resource_ids=None` It is recommended to put the new parameters after the existing parameter of the method

Reply:
`--skip-all-parameterization` and `--skip-resource-name-parameterization` are too long for the customers to input. Could you make them shorter?

Reply:
How about --skip-all-params and --skip-resource-name-params?